src/ui/full-ui.component.js

```diff
@@ -45,6 +45,7 @@ export default class FullUI extends Component {
     return (
       <div>
         <button
+          part={this.atLeastOneOverride() ? "button-with-override" : "button-no-override"}
```

Comment:
Thinking this over more, I don't think changing the `part` based on override status is the right way to do this, since it's the same element just in a different state. Instead, would the following work?

```css
/* When there is an override */
import-map-overrides-full::part(button).imo-current-override {
}

/* When there is not an override */
import-map-overrides-full::part(button):not(.imo-current-override) {
}
```

```suggestion
          part="button"
```